book/accessibility.md

```diff
@@ -1135,7 +1136,7 @@ If you print out `focusable_nodes` for the
 [focus example](examples/example14-focus.html), you should
 get this:
 
-``` {.python .example}
+``` {.python .output}
```

Comment:
I don't fully understand the change from example to output. What impact does it have on the rendered book?

Reply:
None, I found it helpful. `example` is code, `output` isn't code. If you don't like it I can make them all `example` but I sometimes find myself asking "is this really example code" and often that's because it's actually output.